### ISSUE TITLE
Wrong rendering after opening external overlay window

### ISSUE THREAD

**Issue by stefaniapedrazzi** [cyberbotics/webots#623]
After loading many worlds, at some time the rendering becomes much darker than what it should be.
For example starting Webots and running the Guided Tour, at some point I got this appearance for the `receiver_noise.wbt` simulation:

![receiver_noise](https://user-images.githubusercontent.com/5910449/59827604-719c5300-9339-11e9-8294-5bba8a31c49d.png)

Then, I quitted and restarted Webots and the rendering was correct:

![receiver_noise_1](https://user-images.githubusercontent.com/5910449/59827612-76f99d80-9339-11e9-95ba-0884875e07e1.png)

I could reproduce this behaviour a couple of times (at some point the also the hokuyo.wbt Display overlays were not shown any longer on the scene).

Probably my NVIDIA drivers are not up-to-date, but it would be strange if this is the cause of the issue because it works at the beginning.

System: Windows 10
Graphics card: NVIDIA GeForce GT 640M with 2GB VRAM
OpenGL version: 3.3.0 NVIDIA 418.81
Ambient Occlusion: Medium
Texture Quality: High

**Comment by omichel** [comment]
I can't reproduce this on my system:
```
$ webots --sysinfo
System: Windows 10 64-bit
Processor: Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
Number of cores: 4
OpenGL vendor: NVIDIA Corporation (0x10de)
OpenGL renderer: GeForce GTX 970/PCIe/SSE2 (0x13c2)
OpenGL version: 4.6.0 NVIDIA 388.13
```
However, I noticed twice a random crash when running the `animated_skin.wbt` simulation (5th simulation of the guided tour). Can you reproduce the problem you experienced if you don't load the `animated_skin.wbt` before? If so, I would suspect a corruption related to this simulation...

**Comment by stefaniapedrazzi** [comment]
Yes, I could reproduce it just running the devices worlds.

**Comment by omichel** [comment]
And can you reproduce it by simply reverting many times `receiver_noise.wbt`?

**Comment by stefaniapedrazzi** [comment]
I will check.
When I took the screenshots I started from the gps.wbt device worlds (because I just had the same issue with hokuyo.wbt)
I'm sure that it the problem is not in the `receiver_noise.wbt` world, because if I opened other worlds after it I had the same effect. But it would of course be good to know if there is a specific world causing issues and `receiver_noise.wbt` is simple enough to exclude basic issues.

**Comment by stefaniapedrazzi** [comment]
I tried but I cannot reproduce it, neither reverting the `receiver_noise.wbt` nor running other devices worlds.

**Comment by stefaniapedrazzi** [comment]
I will close this issue as I could not reproduce it since a while.

**Comment by stefaniapedrazzi** [comment]
I could reproduce it again when testing Webots R2020a in a VM (low texture quality, disabled shadows and disabled ambient occlusion and disabled anti-aliasing).
I was running the devices guided tour and the problem occurred again when loading the `receiver_noise.wbt` (after `range_finder.wbt`).
In the console I got many errors:
```
OpenGL error: GL_INVALID_OPERATION
```

I tried to reload or switch to other worlds, but the rendering was only fixed after restarting Webots.
